ui/app/templates/vault/cluster.hbs

```diff
@@ -3,37 +3,37 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 <Sidebar::Nav::Cluster />
-{{#if this.auth.currentToken}}
-  {{#if this.customMessages.bannerMessages.length}}
-    {{#each this.customMessages.bannerMessages as |bannerMessage|}}
-      <Hds::Alert @type="inline" @color="warning" data-test-alert={{bannerMessage.id}} as |A|>
-        <A.Title data-test-alert-title={{bannerMessage.id}}>{{bannerMessage.title}}</A.Title>
-        <A.Description data-test-alert-description={{bannerMessage.id}}>
-          {{bannerMessage.message}}
-        </A.Description>
-        <A.Description class="has-top-margin-xs">
-          {{! TODO: VAULT-22908 display links when api is updated to { link: { 'learn': 'www.learn.com'} } }}
-        </A.Description>
-      </Hds::Alert>
-    {{/each}}
-  {{/if}}
-  {{#if this.customMessages.modalMessages.length}}
-    {{#each this.customMessages.modalMessages as |modalMessage|}}
-      <Hds::Modal id={{modalMessage.id}} @size="large" @color="warning" data-test-modal={{modalMessage.id}} as |M|>
-        <M.Header data-test-modal-title={{modalMessage.id}}>
-          {{modalMessage.title}}
-        </M.Header>
-        <M.Body data-test-modal-body={{modalMessage.id}}>
-          {{modalMessage.message}}
-          {{! TODO: VAULT-22908 display links when api is updated to { link: { 'learn': 'www.learn.com'} } }}
-        </M.Body>
-        <M.Footer as |F|>
-          <Hds::Button @text="Confirm" {{on "click" F.close}} data-test-modal-button={{modalMessage.id}} />
-        </M.Footer>
-      </Hds::Modal>
-    {{/each}}
-  {{/if}}
-{{/if}}
+{{#each this.customMessages.bannerMessages as |bannerMessage|}}
+  <Hds::Alert @type="inline" @color="warning" data-test-alert={{bannerMessage.id}} as |A|>
+    <A.Title data-test-alert-title={{bannerMessage.id}}>{{bannerMessage.title}}</A.Title>
+    <A.Description data-test-alert-description={{bannerMessage.id}}>
+      {{bannerMessage.message}}
+      {{#each-in bannerMessage.link as |title href|}}
+        {{#if (and title href)}}
```

Comment:
I don't believe this conditional is necessary since there will be nothing to iterate over if the object is empty

Reply:
That's what I was thinking too, but ran into an error where I was passing undefined to href for <Hds::Link::Inline />

`<Hds::Link::Inline @href={{href}}>{{title}}</Hds::Link::Inline>`

Reply:
Ah yeah - I wondered if the HDS component would error

Another option is wrapping the `{{#each-in}}` in a conditional to avoid iterating all together (in my PR which I'm still working on tests for 😭 I moved the `is-empty-value` template helper to the core addon
```hbs
{{#unless (is-empty-value bannerMessage.link}}
 {{#each-in bannerMessage.link as |title href|}}

Reply:
Awesome I will try that!